server/libs/userlibs.js

```diff
@@ -72,30 +89,26 @@ const logIn = function(body) {
                 }
             })
         }).catch((error) => {
-            reject({
-                status : statusCodes.badRequest,
-                message : `Something ${messages.notMatch}`,
-                error
-            });
+            reject(error);
         })
     })
 };
 
 const updateUser = function (body) {
     let result = fieldsValidate(body);
-
     return new Promise((resolve,reject) => {
         if (result) {
```

Comment:
same as above

Reply:
I am using result variable after if to send as message

Reply:
change the name